velox/exec/HashTable.cpp

```diff
@@ -637,11 +637,12 @@ void HashTable<ignoreNullKeys>::clear() {
 template <bool ignoreNullKeys>
 void HashTable<ignoreNullKeys>::checkSize(int32_t numNew) {
   VELOX_CHECK(
-      size_ == 0 || size_ > (numDistinct_ + numTombstones_),
-      "size_:{} numDistinct_:{} numTombstoneRows_:{}",
+      size_ == 0 || size_ >= (numDistinct_ + numTombstones_),
+      "size_:{} numDistinct_:{} numTombstoneRows_:{} hashMode:{}",
```

Comment:
Let's add commas after each "{}" (except the last one) and remove underscores and colons, so it looks like (more readable):
`size 131072, numDistinct 131072, numTombstoneRows 0, hashMode SomeMode`
instead of 
`size_:131072 numDistinct_:131072 numTombstoneRows_:0 hashMode:SomeMode`